### ISSUE TITLE
[Doc]: Get-AzSqlDatabaseTransparentDataEncryptionActivity in Az.Sql was removed in az 7.4.0 but doc and examples still exists in Az 10.0.0

### ISSUE THREAD

**Issue by VF-atempel** [Azure/azure-powershell#22062]
### Description

Hello,

categorie : Cmdlet reference doc: https://docs.microsoft.com/en-us/powershell/module

May I ask why this cmdlet (modul az.sql, Get-AzSqlDatabaseTransparentDataEncryptionActivity) is still listed and a documentation is to be found with examples?

Link to source:
https://learn.microsoft.com/en-us/powershell/module/az.sql/get-azsqldatabasetransparentdataencryptionactivity?view=azps-10.0.0

If I am not mistaken it is removed since az 7.4.0 - at least there is to be found this release news:
[Breaking change] Removed 'Get-AzSqlDatabaseTransparentDataEncryptionActivity'
https://learn.microsoft.com/en-us/powershell/azure/release-notes-azureps?view=azps-10.0.0#azsql-6
[Cmdlet reference doc: https://docs.microsoft.com/en-us/powershell/module](https://learn.microsoft.com/en-us/powershell/azure/release-notes-azureps?view=azps-10.0.0#740---april-2022)

In addition to it may I ask if there is an replacement.

I was a little confused about it.

Feedback is welcome.

Thanks in advence
Alex

**Comment by isra-fel** [comment]
Thanks Alex for reporting!
@BethanyZhou can you double confirm? And if that's the case, please also check if there are other reference docs of removed cmdlets and how to make sure we don't publish them in the future.

**Comment by VF-atempel** [comment]
Hello,
thanks for coming back so fast.

I had double checked it today. It is like I describe it above.

In addition to it I have some more tech data to share about it.

`
$PSVersionTable.PSVersion
Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      3      4

Get-Module -ListAvailable -Name Az -Refresh
ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     10.0.0                Az                                  Core,Desk


Get-Module Az.Sql
ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     4.7.0                 Az.Sql                              {Add-AzSqlDatabaseToFailoverGroup, Add-AzSqlElasticJobStep, Add-AzSqlElasticJobTarget, Add-AzSqlInstanceKeyVaultKey…}

If you run code like in the docs (example) you receive this error.

> $databaseTdeActivity = Get-AzSqlDatabaseTransparentDataEncryptionActivity -ServerName $paasSqlServerName  -ResourceGroupName $resourceGroupName -DatabaseName $paasSqlDatabaseName
$databaseTdeActivity
WARNING: The command Get-AzSqlDatabaseTransparentDataEncryptionActivity is part of Azure PowerShell module "Az.Sql" and it is not installed. Run "Install-Module Az.Sql" to install it.
Get-AzSqlDatabaseTransparentDataEncryptionActivity: The term 'Get-AzSqlDatabaseTransparentDataEncryptionActivity' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
`

If you have a look into the current az.sql (4.7.0) posh module, you see that it actually get still exportet.

![image](https://github.com/Azure/azure-powershell/assets/136493659/14a53c86-ce07-4370-9e47-62231b319ab2)

Guessing this cmdlet is no longer in the dll.

Looks like some minor inconsistencies.

Let me know if you need additional info.

Thanks

**Comment by BethanyZhou** [comment]
Yes, this cmdlet was removed from Az.Sql>=3.8.0 https://github.com/Azure/azure-powershell/blob/main/src/Sql/Sql/ChangeLog.md#version-380. But its help markdown and definition are kept...

Removed them by PR: https://github.com/Azure/azure-powershell/pull/22077

Also checked other removed cmdlets. No removed cmdlet's files/information are detected luckily.

>  consider how to ensure we don't repeat our mistakes in the future
I think that's the gap of our example analyzer because we only analyze touched/modified files in current PR. How about we set a pipeline to scan all markdowns regularly so that our example analyzer can detect this cmdlet is not valid? @isra-fel
